content/version/2/0/code_of_conduct.sw.md

```diff
@@ -0,0 +1,83 @@
+* Kuwajibika na kuomba msamaha kwa wale walioathiriwa na makosa yetu, na kujifunza kutokana na hayo
+* Kuzingatia yale ambayo sio bora kwetu tu kibinafsi, bali pia jamii kijumla
+
+Zifuatazo ni mifano ya tabia zisizokubalika:
+
+* Matumizi ya taswira au lugha ya kujamiiana, na unyanyasaji wowote wa kijinsia
+* Kufuatiliana kwa kiwango kisicho cha kawaida, kutukanana au kutoa maoni ya dharau, na mashambulizi ya kibinafsi au ya kisiasa
+* Unyanyasaji hadharani au kwa faragha
+* Kuchapisha habari za kibinafsi za wengine, kama vile makazi ya kudumu na barua pepe, bila idhini yao wazi
+* Mienendo mengine ambayo inaweza kuchukuliwa kuwa haifai katika mpangilio wa kitaaluma
+
+## Wajibu wa Utekelezaji
+
+Viongozi wa jumuiya wanawajibika kubainisha na kutekeleza viwango vyetu vya tabia zinazokubalika na watachukua hatua zinazofaa na sahihi za marekebisho, kukabiliana na tabia yoyote wanayofikiri haifai, inatisha, inakera, au ina madhara.
+
+Viongozi wa jumuiya wana haki na wajibu wa kuondoa, kubadilisha, au kukataa maoni, mabadiliko, programu, masahihisho ya wiki, maswala, na michango mingine ambayo haijaambatana na kanuni hizi za maadili, na kuwasiliana kuhusu sababu za kudhibitisha maamuzi wakati muafaka.
+
+## Wigo wa Matumizi
+
+Kanuni ya maadili mema inatumika katika maeneo yote ya jamii nzima, na pia inatumika wakati mtu binafsi anayewakilisha jamii kirasmi katika maeneo ya umma. Mifano ya kuwakilisha jamii yetu ni kutumia barua pepe, kuchapisha kwa kutumia vyombo rasmi vya habari kwenye akaunti ya jumuiya, au kufanya kazi kama kuteuliwa mwakilishi kwenye mitandao au nje ya mitandao.
+
+## Utekelezaji
+
+Matukio ya matusi, unyanyasaji au tabia nyingine isiyokubalika, inaweza ripotiwa kwa viongozi wa jamii wanaohusikana utekelezaji katika [WEKA MBINU YA MAWASILIANO]. Malalamiko yote yatapitiwa na kuchunguzwa mara moja na kwa haki.Viongozi wote wa jamii wana wajibu wa kuheshimu faragha na usalama wa mwandishi wa tukio lolote.
+
+## Mwongozo wa Utekelezaji
+
+Viongozi wa jumuiya watafuata Miongozo hii ya athari ya jamii katika kuamua madhara ya hatua yoyote wanayoona inakiuka kanuni hizi za maadili:
+
+### 1. Marekebisho
+
+**Athari ya Jamii**: Matumizi ya lugha isiyofaa au tabia nyingine inayoonekana isiyo ya kitaaluma au isiyokubalika katika jamii.
+
+**Matokeo**: Onyo la kibinafsi, lililoandikwa kutoka kwa viongozi wa jamii, linalotoa uwazi karibu na asili ya ukiukaji na maelezo ya kwa nini tabia si muafaka. Msamaha kwa umma unaweza kuombwa.
+
+### 2. Onyo
+
+**Athari ya Jamii**: Ukiukaji kupitia tukio moja au mfululizo ya vitendo.
+
+**Matokeo**: Onyo kwa matokeo ya tabia inayozidi. Hapana mwingiliano na watu waliohusika, ikiwa ni pamoja na mwingiliano usiotarajiwa na wale wanaotimiza Kanuni za Maadili, kwa muda maalum. Hii ni pamoja na kuepuka mwingiliano katika nafasi za jamii na vile vile njia za nje kama mitandao ya jamii. Kukiuka sheria hizi kunaweza kusababisha marufuku ya muda fulani au ya kudumu.
```

Comment:
Could we change this statement from `Hapana mwingiliano na watu waliohusika,..` to ` mwingiliano na watu waliohusika haurusiwi,..`